PiwikTracker/PiwikTracker.swift

```diff
@@ -96,6 +96,7 @@ final public class PiwikTracker: NSObject {
             guard events.count > 0 else {
                 // there are no more events queued, finish dispatching
                 self.isDispatching = false
+                self.startDispatchTimer()
```

Comment:
Maybe it is possible to cleanup the whole dispatching process. It looks a bit overly complex for what it does. I will try to create a proposal.